client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Review page
![Screenshot of Review page showing 10px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/bcea3512-53ff-42c3-8803-8ebb94f97e02)

Reply:
These tooltips still appear as 10px.

Reply:
This should be 16px now